R/thin_draws.R

```diff
@@ -4,8 +4,11 @@
 #'
 #' @aliases thin
 #' @template args-methods-x
-#' @param thin (positive integer) The period for selecting draws.
+#' @param thin (positive integer) The period for selecting draws. If
+#'   omitted, this will be automatically calculated based on bulk-ESS
```

Comment:
Minor nitpick, this should say `NULL` not "omitted" (which could be taken to mean a missing arg):

```suggestion
#'   `NULL`, this will be automatically calculated based on bulk-ESS
```